README.md

```diff
@@ -152,14 +152,9 @@ The fourth byte from the end in the received data (0x03 in this example) will te
 If you would like to use a your own font for OpenDPS you may do so by doing the following:
 
 ```
-% cd opendps
-% python generate_font.py -f gfx/Ubuntu-C.ttf
-% make clean
-% make
+% make -C opendps fonts FONT_FILE=<path_to_font> FONT_SMALL_SIZE=18 FONT_MEDIUM_SIZE=24 FONT_LARGE_SIZE=48
```

Comment:
Perhaps it should be mentioned that the generator supports TTF fonts.

Reply:
I'll add this info. It also supports otf